.gitlab-ci.yml

```diff
@@ -30,6 +30,27 @@ test-features:
   script:
     - cargo test --all --all-features --all-targets --locked
 
+integration-tests:
+  timeout: 5 minutes
+  image: paritytech/ci-unified:bullseye-1.70.0
+  stage: test
+  script:
+    - curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
+    - apt-get install -y nodejs
+    - cargo build -p spiritnet-runtime
+    - cd ./integration-tests/chopsticks
+    - yarn 
+    - yarn test
+
+lint-typescript:
+  timeout: 5 minutes
+  image: node:20
+  stage: test
+  script:
+    - cd ./integration-tests/chopsticks
+    - yarn 
+    - yarn lint
```

Comment:
What about prettier? We don't check for that? Or is that also checked as part of `yarn lint`?

Reply:
It should be tested in yarn lint. I would verify it again.

Reply:
It wasn't checked. Seems like something in the config was wrong. I update the lint command to also check the fmt: [5905280](https://github.com/KILTprotocol/kilt-node/pull/614/commits/59052803d6f9d62b13d657d32f9f3b84c99dd271)